mne/viz/utils.py

```diff
@@ -2360,29 +2360,12 @@ def _make_combine_callable(combine):
 def _convert_psds(
     psds, dB, estimate, scaling, unit, ch_names=None, first_dim="channel"
 ):
-    """Convert PSDs to dB (if necessary) and appropriate units.
-
-    The following table summarizes the relationship between the value of
-    parameters ``dB`` and ``estimate``, and the type of plot and corresponding
-    units.
-
-    | dB    | estimate    | plot | units             |
-    |-------+-------------+------+-------------------|
-    | True  | 'power'     | PSD  | amp**2/Hz (dB)    |
-    | True  | 'amplitude' | ASD  | amp/sqrt(Hz) (dB) |
-    | True  | 'auto'      | PSD  | amp**2/Hz (dB)    |
-    | False | 'power'     | PSD  | amp**2/Hz         |
-    | False | 'amplitude' | ASD  | amp/sqrt(Hz)      |
-    | False | 'auto'      | ASD  | amp/sqrt(Hz)      |
```

Comment:
why remove the whole table, instead of just the "auto" lines?

Reply:
I don't think this table is useful after "auto" is gone, since there is no interaction between dB and estimate. The units and type of spectrum are now part of the y-axis label anyway.

Reply:
Fair enough.